css/properties/animation-duration.json

```
@@ -115,6 +115,42 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "auto": {
+          "__compat": {
+            "description": "The <code>auto</code> value was proposed as part of the <a href='https://drafts.csswg.org/css-animations-2/#propdef-animation-duration'>CSS Animations Level 2</a> specification and currently has limited support.",
```

The description is way too long. It meant more like a label.

```suggestion
            "description": "<code>auto</code> value",
```